README.md

```diff
@@ -16,24 +19,43 @@ practices with Custom Resources and Kubernetes-native primitives.
 
 ---
 
-[Documentation](#documentation) | [Learning with Videos](#learning-with-videos) | [Quickstart Examples](#quickstart-examples)
-| [IDE Integration](#ide-integration) | [Community](#community)
+[Project Status](#project-status) | [Introduction](#introduction) | [Documentation](#documentation) | [Learning with Videos](#learning-with-videos) | [Quickstart Examples](#quickstart-examples) | [Community](#community)
 
 ---
 
-## Documentation
+## Project Status
+Tyk Operator is currently in beta. We follow the [Kubernetes definition of beta](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages). We welcome users to try it out and give us feedback [here](https://github.com/TykTechnologies/tyk-operator/discussions) or via support channel for existing Tyk customers.
+
+
+## Introduction
+
+### What can you do with Tyk Operator?
 
-Read more about the [concepts here](./docs/concepts.md).
+Tyk Operator can configure Tyk Gateway as a drop-in replacement for standard [Kubernetes Ingress](./docs/ingress.md). You can manage your API definitions and security policies with it. It also works with the classic portal so you can manage your Classic Portal declaratively.
 
-Custom Tyk Objects are available as [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-and documentation for each of these custom resources are available:
+Custom Tyk Objects are available as [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and documentation for each of these custom resources are available [here](./docs/crd.md).
 
-- [Installing Tyk Operator](./docs/installation/installation.md)
-- [Getting Started](./docs/getting-started.md)
-- [Ingress Controller](./docs/ingress.md)
-- [API Definitions](./docs/api_definitions.md)
-- [Security Policies](./docs/policies.md)
-- [Multi Gateway with Operator Context](./docs/operator_context.md)
+
+### Why use Tyk Operator?
+
+You can get the benefits of GitOps with declarative API configurations:
+
+- **Security and Compliance**: All changes must go through peer review through pull requests. The configurations are versioned in your version control system and approved by your API Product Owner and Platform team.
+
+- **Kubernetes-Native Developer Experience**: API Developers enjoy a smoother Continuous Integration process as they can develop, test, and deploy the microservices and API configurations together using familiar development toolings and pipeline.
+
+- **Reliability**: With declarative API configurations, you have a single source of truth to recover after any system failures, reducing the meantime to recovery from hours to minutes.
+
+
+
+## Documentation
```

Comment:
Just an idea; what if we mention example yaml files, that we created under [samples](https://github.com/TykTechnologies/tyk-operator/tree/master/config/samples) folder, here?